src/paymentservice/index.js

```diff
@@ -16,19 +16,43 @@
 
 'use strict';
 
-require('@google-cloud/profiler').start({
-  serviceContext: {
-    service: 'paymentservice',
-    version: '1.0.0'
-  }
-});
-require('@google-cloud/trace-agent').start();
-require('@google-cloud/debug-agent').start({
-  serviceContext: {
-    service: 'paymentservice',
-    version: 'VERSION'
-  }
-});
+
+if(process.env.DISABLE_PROFILER) {
+  console.log("Profiler disabled.")
```

Comment:
Suggestion: Should we use `pino` instead of `console.log`?
* I'm guessing we copied this newly added code from [`/src/currencyservice/server.js`](https://github.com/GoogleCloudPlatform/microservices-demo/blob/2554fe8b154750e17fa8a5a3bb7dd402489d13a7/src/currencyservice/server.js).
* We could define a `pino` logger at the top of `src/paymentservice/index.js` (after `'use strict';`) and use that instead:
```
const logger = pino({
  name: 'paymentservice-server',
  messageKey: 'message',
  changeLevelName: 'severity',
  useLevelLabels: true
});
```
* We could also refactor [`/src/currencyservice/server.js`](https://github.com/GoogleCloudPlatform/microservices-demo/blob/2554fe8b154750e17fa8a5a3bb7dd402489d13a7/src/currencyservice/server.js) in a similar fashion.
  * It looks like the use of `pino` was introduced in [#66](https://github.com/GoogleCloudPlatform/microservices-demo/pull/66). 
  * And the console.log statements were later introduced in [#293](https://github.com/GoogleCloudPlatform/microservices-demo/pull/293).
  * So the use of `console.log` _could_ _possibly_ be intentional?

Reply:
Yes I copied this from currencyservice - is this in scope for this PR, or can we add the `pino` logging in another PR?

Reply:
Agreed. The `pino`-related work is outside the scope of this PR.
I've created an issue to track `pino`-related work: https://github.com/GoogleCloudPlatform/microservices-demo/issues/674.